### PR TITLE
Add link to Cricinfo

### PR DESCRIPTION
**Pull request by raksh93**
@casey-collab

### CODE REVIEW

File: english/play-some-cricket/playsomecricket.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
 
 [Mr. Cool Dhoni finishes off.](https://youtu.be/wbfv1qOG3WM)
 
+[Follow cricket news](english/cricket/cricket.md)
```
Comment (casey-collab) [inline english/play-some-cricket/playsomecricket.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/cricket/cricket.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Click [here](http://www.espncricinfo.com) to catch up on cricket news and view live scores.
```
Comment (casey-collab) [inline english/cricket/cricket.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by raksh93** [comment]
@casey-collab

File: english/play-some-cricket/playsomecricket.md [hunk 3]
```diff
@@ -4,3 +4,4 @@
 
 [Mr. Cool Dhoni finishes off.](https://youtu.be/wbfv1qOG3WM)
 
+[Follow cricket news](~/english/cricket/cricket.md)
```
Comment (casey-collab) [inline english/play-some-cricket/playsomecricket.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/cricket/cricket.md (hunk 2 above)
Comment (casey-collab) [inline english/cricket/cricket.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by raksh93** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :hibiscus:

**Comment by raksh93** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request merged**